web/src/api/images/verify_app.ts

```diff
@@ -0,0 +1,278 @@
+import {
+  errorHasuraQuery,
+  errorNotAllowed,
+  errorResponse,
+} from "src/backend/errors";
+import { NextApiRequest, NextApiResponse } from "next";
+import { protectInternalEndpoint } from "src/backend/utils";
+import { getAPIReviewerGraphqlClient } from "src/backend/graphql";
+import { getSdk as getAppMetadataSDK } from "@/api/images/graphql/getAppMetadata.generated";
+import { getSdk as deleteVerifiedAppMetadataSDK } from "@/api/images/graphql/deleteVerifiedAppMetadata.generated";
+import { getSdk as updateAppMetadataSDK } from "@/api/images/graphql/updateAppMetadata.generated";
+import { getFileExtension } from "src/backend/utils";
+
+import {
+  S3Client,
+  ListObjectsCommand,
+  PutObjectTaggingCommand,
+  CopyObjectCommand,
+} from "@aws-sdk/client-s3";
+import { logger } from "@/lib/logger";
+import { randomUUID } from "crypto";
+
+export type VerifyAppResponse = {
+  success?: boolean;
+};
+
+/**
+ * This function handles the verification of the app by a reviewer
+ * @param req Expects an app_id and reviewer_name is_reviewer_app_store_approved and is_reviewer_world_app_approved
+ * @param res Returns a success message if the app was verified
+ */
+export const handleVerifyApp = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+) => {
+  try {
```

Comment:
let's avoid the global try/catch, either we're returning a handled response or for unhandled errors will be logged and hasura will handle the 500